src/Pages/List/Availables/Availables.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import ImgSlider from '../../../Components/ImgSlider/ImgSlider';
+import PaginationBtns from './PaginationBtns/PaginationBtns';
+import './Availables.scss';
+
+const NUMBER_FORMAT_REGEXP = /\B(?=(\d{3})+(?!\d))/g;
+
+class Availables extends Component {
+
+
+  goToDetail = (id) => {
+    this.props.history.push(`/details/${id}`);
+  }
+
+  render() {
+    
+    const {searchOptions, rooms} = this.props;
+    console.log(searchOptions, "availables")
+    return (
```

Comment:
```suggestion
  render() {    
    const {searchOptions, rooms} = this.props;

    return (
```
- 테스트 끝난 콘솔은 반드시 제거해주세요
- 불필요한 개행 없애주세요

Reply:
넵